nextjs-13/src/components/PublicLayout/index.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react';
+import { useSession } from 'next-auth/react';
+
+import PublicLayout from './';
+
+jest.mock('next-auth/react');
+jest.mock('next/navigation');
+
+describe('PublicLayout', () => {
+  it('renders the component', () => {
```

Comment:
This could be moved in the context below at line 36.

Reply:
Did you mean that we should move the line `jest.mock('next/navigation')` or the `it('renders the component'`? 👀

Btw, I just realized that the `jest.mock('next/navigation')` is redundant here, let me remove it first!

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/127/commits/9f6a6ba6b4e3b911041ca4a9ec1d66946c0d280f)